### PR TITLE
[#XGB-202] Write section development setup

### PR DESCRIPTION
**Pull request by raphiz**

### CODE REVIEW

File: documents/final-submission-document/chapters/4_implementation.tex [hunk 1]
```diff
@@ -4,12 +4,31 @@ \chapter{Implementation and Testing} % Realisierung und Test
...
 
 \section{Development Setup}
-% - e2e tests
-% - angular cli
-% - build script / ci
-% - docker-compose, CA automated with scripts etc. SE stuff
 
-\section{Encountered Problems}
+Figure \fullref{fig:development-setup} illustrates the development setup in the form of an UML deployment diagram.
+A developer connects via his browser to the reverse proxy that serves the XMPP-Grid Broker web application.
```
Comment (fabianhauser) [inline documents/final-submission-document/chapters/4_implementation.tex]: I think using "reverse proxy" here can be a bit confusing, as it actually is a web server. I suggest changing it to that, do you agree?

File: documents/final-submission-document/chapters/4_implementation.tex [hunk 2]
```diff
@@ -4,12 +4,31 @@ \chapter{Implementation and Testing} % Realisierung und Test
 
 
 \section{Development Setup}
-% - e2e tests
-% - angular cli
-% - build script / ci
-% - docker-compose, CA automated with scripts etc. SE stuff
 
-\section{Encountered Problems}
+Figure \fullref{fig:development-setup} illustrates the development setup in the form of an UML deployment diagram.
```
[the remark this answers is not in the document]
Comment (raphiz) [reply documents/final-submission-document/chapters/4_implementation.tex]: ah, sure :laughing:

File: documents/final-submission-document/chapters/4_implementation.tex (hunk 1 above)
Comment (raphiz) [reply documents/final-submission-document/chapters/4_implementation.tex]: We leave it that way (as discussed) :crying_cat_face: :disappointed:

**Comment by raphiz** [comment]
Good Work!

**Pull request merged**